types/simulation/types.go

```diff
@@ -64,7 +75,7 @@ type WeightedOperation interface {
 //
 // Operations can optionally provide a list of "FutureOperations" to run later
 // These will be ran at the beginning of the corresponding block.
-type Operation func(r *rand.Rand, app *baseapp.BaseApp,
+type Operation func(r *rand.Rand, app AppEntrypoint,
```

Comment:
This is a breaking change that would affect module developers. The main motivation was to remove the dependency to baseapp for the x/modules.
I will 🤔 about some alternatives

Reply:
For now, we don't need to do this.  I think we can just keep the baseapp dependency